buildenv.sh

```diff
@@ -32,14 +94,30 @@ TOOLS_DIR="$OUT_DIR/tools/bin"
 
 PATH="$TOOLS_DIR:$PATH"
 
-run_cmd()
+unica()
 {
     local CMD=$1
     local CMDS
     CMDS="$(find "scripts" -mindepth 1 -maxdepth 1 -type f -printf '%f\n' | sed "s/.sh//g")"
 
+    local YELLOW="\033[1;33m"
+    local CYAN="\033[1;36m"
+    local RESET_COLOR="\033[0m"
+
     if [ -z "$CMD" ] || [ "$CMD" = "-h" ]; then
-        echo -e "Available cmds:\n$CMDS"
+        echo -e "Available cmds:"
+        for script in "scripts"/*.sh; do
+            CMD_NAME=$(basename "$script" .sh)
+                        HELP_LINES=$(grep "^# CMD_HELP" "$script" | sed 's/^# CMD_HELP[[:space:]]*//')
```

Comment:
Just a thought of mine, I think creating a function inside each script which prints the help is a bit better than this actual implementation.